minidump-stackwalk/Cargo.toml

```diff
@@ -38,6 +38,5 @@ test-assembler = "0.1.6"
 
 [package.metadata.release]
 pre-release-replacements = [
-  {file="../RELEASES.md", search="Next Version", replace="Version {{version}} ({{date}})", min=1},
-  {file="../RELEASES.md", search="<!-- next-header -->", replace="<!-- next-header -->\n# Next Version\n", exactly=1},
-]
+  {file="../CHANGELOG.md", search="# Unreleased", replace="# Unreleased\n\nNothing Yet!\n\n\n# Version {{version}} ({{date}})", exactly=1},
```

Comment:
This should still be `../RELEASES.md` right? Or did you want to rename `RELEASES.md` to `CHANGELOG.md`?